eclipse-che/che-operator
# feat: update traefik to 2.5.0

Signed-off-by: Michal Vala <mvala@redhat.com>



### What does this PR do?
update traefik to 2.5.0

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/20142


### How to test this PR?


Prerequisite: use template from PR branch -> clone `https://github.com/sparkoo/che-operator/tree/gh20142-updateTraefik` and build the templates https://github.com/eclipse-che/che-operator#deploy-che-operator-with-chectl-using---installer-operator-flag. Then always deploy with these templates.

1. deploy Che with `server.serverExposureStrategy: single-host`
2. check that traefik container in gateway is 2.5.0
3. start a workspace

_

1. deploy Che with devworkspaces enabled
2. check that traefik container in gateway is 2.5.0
3. start a workspace


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
